packages/xml-views-validation/src/utils/deprecated-message-builder.ts

```diff
@@ -0,0 +1,23 @@
+import { UI5DeprecatedInfo } from "@ui5-language-assistant/semantic-model-types";
+
+// TODO: unit tests for all edge cases
+export function buildDeprecatedIssueMessage({
+  ui5Kind,
+  fqn,
+  deprecatedInfo
+}: {
+  // TODO: add more kinds here as needed (e.g property/aggregation/...)
+  ui5Kind: "Class";
+  fqn: string;
+  deprecatedInfo: UI5DeprecatedInfo;
+}): string {
+  const msgPrefix = `UI5 ${ui5Kind}: ${fqn} is deprecated`;
+  const sinceOptionalPart = deprecatedInfo.since
+    ? ` since: ${deprecatedInfo.since}`
```

Comment:
In the documentation I use `since version <since>` without the colon (they don't write "version" in the `@since` tag)

Reply:
I based the string on the previous error message (with corrections).
- https://github.wdf.sap.corp/DevX/xmlvalidator/blob/master/src/fioriAnalysis/rules/XML_DEPRECATION.js#L79-L83

I don't have a strong opinion either way, lets let UX decide on this when we resolve: https://sapjira.wdf.sap.corp/browse/DEVXKINGSTON-937

Reply:
It didn't have `:` and it said "version"... it was `The control <fqn> is deprecated[ since version <since>].[\n\t<description>]` (where `[]` parts are optional)

Reply:
I honestly have difficulty noticing these small differences, Lets leave this for proper PR made by UX folk to decide on the ultimate error string format.

Reply:
The diff is: instead of 
`` ` since: ${deprecatedInfo.since}` ``
Use 
`` ` since version ${deprecatedInfo.since}` ``

Reply:
Ah o.k. I will see if I can change this before UX review.